APIs/schemas/auth_metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Authorization API metadata resource",
+  "description": "Displays the Authorization server metadata",
+  "properties": {
```

Comment:
NOTE: this may need updating with the REQUIRED fields from Section 2 of https://tools.ietf.org/html/rfc8414